templates/tableau-single-server-master.template

```diff
@@ -278,20 +276,7 @@
                                     "QuickStartS3URL"
                                 ]
                             },
-                            {
-                                "Fn::FindInMap": [
-                                    "DefaultConfiguration",
-                                    "QSS3Config",
-                                    "QSS3BucketName"
-                                ]
-                            },
-                            {
-                                "Fn::FindInMap": [
-                                    "DefaultConfiguration",
-                                    "QSS3Config",
-                                    "QSS3KeyPrefix"
-                                ]
-                            },
+                            "quickstart-reference/tableau",
```

Comment:
This needs to be exposed at a parameter (that is, QSS3BucketName and QSS3KeyPrefix). For example on this, please see: https://github.com/aws-quickstart/quickstart-examples/blob/master/templates/workload-json-master.template

Reply:
Also, the value of QSS3KeyPrefix should be: "tableau/server/latest/"

Reply:
In your last review, you suggested " Note: Please don’t include the QSS3BucketName and QSS3KeyPrefix parameters in those .json parameter files when you are making the updates." Should I make QSS3BucketName and QSS3KeyPrefix as parameters?

Reply:
Yes, we still want to have them in the template as parameter if they're to be used internally (which they are due to the nested stack/template). I should've been more clear about the .json parameter files... they are used for testing the stacks with "mostly" generic parameters depending on the test scenario. Because the assets of a Quick Start are staged in a location that's different than the production location (i.e. quickstart-reference), we don't include them in the parameters file. To see the existing parameter files in this Quick Start... take a look at the `ci/` directory for the config.yml which specifies the scenarios and the .json files (currently in `templates/`, will very soon move to the `ci/` directory as well).